source/blood/src/common.cpp

```diff
@@ -308,26 +308,15 @@ static int G_ReadRegistryValue(char const * const SubKey, char const * const Val
 static void G_AddSteamPaths(const char *basepath)
 {
     UNREFERENCED_PARAMETER(basepath);
```

Comment:
Should remove this line since this variable is used now.

Reply:
will do